index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href='https://fonts.googleapis.com/css?family=Poppins'>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Document</title>
+</head>
+<body>
+    <!-- navbar -->
+    <header>
+    <nav class="navbar">
+        <a href="#">
+            <h2 class="logo"><strong>My Logo</strong></h2>
+        </a>
+        <nav>
+        <a class="icon">
+            <i class="fa fa-bars"></i>
+        </a>
+        </nav>
+    </nav>
+    </header>
+
+    <!-- headline -->
+    <section class="header">  
+        <h1 class="intro"><b><strong>I'm Martin Glad to see you!</strong></b></h1>
+        <h2 class="presentation">
+            I'm a software developer! I can help you build a product, feature a website. 
+            Look through some of my work and experience!
+            If you like what you see and have a project you need coded, don't hesitate to contact me.
+        </h2>
```

Comment:
:orange_circle: [OPTIONAL]  I think the right tag here is p and not a header tag. This way will be semantically correct and the screen readers and other technologies will thank you